.jenkins/Packaging.Jenkinsfile

```diff
@@ -26,17 +27,22 @@ def packageUpload(String version, String build_type) {
 
 def WindowsUpload() {
```

Comment:
Please add a comment for this function.

Reply:
For now I will decline to add comments for functions, as we haven't done that for our Jenkinsfiles yet. I think we should add some documentation for all of our CI system test cases, though. I know @radhikaj is working on some documentation for supported platforms & configurations.